src/main/scala/io/iohk/ethereum/blockchain/sync/fast/FastSync.scala

```diff
@@ -342,7 +338,7 @@ class FastSync(
         batchFailuresCount += 1
         if (batchFailuresCount > fastSyncMaxBatchRetries) {
           log.info("Max number of allowed failures reached. Switching branch and master peer.")
-          handleRewind(header, masterPeer.get, fastSyncBlockValidationN, blacklistDuration)
+          handleRewind(header, masterPeer.get, fastSyncBlockValidationN, criticalBlacklistDuration)
```

Comment:
According to the comments in base.conf, the `critical-blacklist-duration` should be used in certain cases:
`# Duration for high offense blacklisting of a peer. Blacklisting reason include: header validation failure.`

And I was once being used once

Reply:
👍